### ISSUE TITLE
Docker error message: "ols_litespeed_1 exited with code 1"

### ISSUE THREAD

**Issue by biguenique** [litespeedtech/ols-docker-env#2]
Been running ols-docker-env on a DigitalOcean VPS for a few days without problem.

Today avec restarting the VPS, I was not able to run the OLS service anymore. Running `docker-compose up`, I see that ols_litespeed_1 keeps exiting, issuing error message "ols_litespeed_1 exited with code 1", after a random number of "litespeed_1  | WordPress installation finished." messages.

I see that mysql_1 is running just fine, and I can access Adminer on port 8080. But not litespeed ports (80, 443, 7080). My browser returns ERR_CONNECTION_REFUSED on ports 80 & 443, ERR_CONNECTION_TIME_OUT on port 7080...

I tried `docker-compose pull`, `docker-compose up --build`, but no luck.

Any help would be really welcome!

**Comment by Code-Egg** [comment]
HIHI 
Can you try to access to the litespeedtech container by command `docker exec -it THE_container_ID bash` and run `/usr/local/lsws/bin/lswsctrl status` to check status and then restart process by `/usr/local/lsws/bin/lswsctrl restart` see if it helps?

**Comment by biguenique** [comment]
I tried `docker exec -it ols_litespeed_1 bash` a few times, and got a succesion of different error messages:
- OCI runtime exec failed: exec failed: cannot exec a container that has stopped: unknown
- cannot exec in a stopped state: unknown
- Error response from daemon: Container 3289368055e3cf9e2ee956a822c52139434c6bf3b95987ed05936c2cf2e461d5 is restarting, wait until the container is running

The litespeed container seems to be constantly exiting and restarting. Maybe some log would be relevant to diagnose the problem?

**Comment by biguenique** [comment]
Thanks for the quick answer @Code-Egg by the way!

I managed to fix the problem by restoring the original lsws/admin-conf/admin_config.conf.

Then I was able to bash into the litespeed container using `docker-compose exec litespeed bash`.

The problem seems related to my LetsEncrypt certificates that are gone awreck... These are stored inside the litespeed container right... Seems my /etc/letsencrypt/live folder has gone missing... since the last reboot. The missing certificate seemly caused litespeed to repeatedly crash...

I will try to map /etc/letsencrypt/live to a host directory before re-requesting the certificates to prevent this to happen again.

Thanks again for you quick answer! I'll mark this issue as close as soon as I confirm there is no further issue.
